addon/components/bs-alert.js

```diff
@@ -185,7 +191,7 @@ export default class Alert extends Component {
    * @private
    */
   hide() {
-    if (this.hidden) {
```

Comment:
Reading the tracked property before setting it a few lines below seemed to cause the assertion here.